src/ontology/hp.Makefile

```diff
@@ -636,3 +636,30 @@ $(TMPDIR)/hp-%-merged.owl: hp-base.owl tmp/%.owl
 
 mappings: 
 	$(MAKE_FAST) ../mappings/hp-snomed.lexmatch.sssom.tsv
+
+.PHONY: diff
+diff: diff-table diff-kgcl diff-yaml
+
+tmp/hp-released.db:
+	wget http://purl.obolibrary.org/obo/hp.owl -O tmp/hp-released.owl && cp ../../hp-base.owl tmp/hp-base.owl
+	semsql make tmp/hp-base.db
+	semsql make $@
+
+tmp/hp-released.obo:
+	wget http://purl.obolibrary.org/obo/hp.obo -O tmp/hp-released.obo && cp ../../hp.obo tmp/hp-base.obo
+
+diff-table: tmp/hp-released.db
+	runoak -i sqlite:tmp/hp-base.db diff -X sqlite:tmp/hp-released.db \
+	-o reports/difference_table.tsv --output-type csv --statistics --group-by-property oio:hasOBONamespace
+
+diff-kgcl: tmp/hp-released.db
+	runoak -i sqlite:tmp/hp-base.db diff -X sqlite:tmp/hp-released.db \
+	-o reports/difference_kgcl.txt --output-type kgcl
+
+diff-yaml: tmp/hp-released.db
+	runoak -i sqlite:tmp/hp-base.db diff -X sqlite:tmp/hp-released.db \
+	-o reports/difference_yaml.yaml --output-type yaml
+
+diff-md: tmp/hp-released.obo
+	runoak -i simpleobo:tmp/hp-base.obo diff -X simpleobo:tmp/hp-released.obo \
+	-o reports/difference_md.md --output-type md
```

Comment:
Lets better use this pattern in the makefile:

DIFF_LEFT=released
DIFF_RIGHT=base

```

kgcl-diff-md-release-base: reports/difference_release_base.md

reports/difference_release_base.md: tmp/hp-released.obo tmp/hp-base.obo
	runoak -i simpleobo:tmp/hp-released.obo diff -X simpleobo:tmp/hp-base.obo -o $@ --output-type md
```

Note this is only the "release diff", not useful for the github action. I will make a comment there to explain.

Reply:
It's flipped 
base => left
released => right

Reply:
Done! I had a feeling this would be the case. Makes total sense!